.github/issue_label_bot.yaml

```diff
@@ -1,3 +1,27 @@
+#
```

Comment:
Don't commit this file... looks like I need to add another exception to the license plugin. :)